Database/dbconnect.js

```diff
@@ -0,0 +1,13 @@
+const mongoose = require('mongoose');
+
+mongoose.set('strictQuery', false);
+mongoose.connect("mongodb://localhost:27017/KYC_Details",{
```

Comment:
`mongodb://localhost:27017`
This should go in `.env` file.

Reply:
Done with the changes